hammer_cli_reference/playbook.yaml

```diff
@@ -0,0 +1,100 @@
+---
+- name: "Check prerequisites on developer-system"
+  connection: local
+  hosts: developer-system
+  vars:
+    repo_path: "/home/user"
+    repo_remote: "git@github.com:theforeman/foreman-documentation.git"
+
+  tasks:
+    - name: "Verify that user input is overwritten"
+      ansible.builtin.assert:
+        that: repo_path != "/home/user"
+
+    - name: "Verify that shellscript is present"
+      ansible.builtin.stat:
+        path: "{{ repo_path }}/guides/scripts/generate-hammer-reference.sh"
+      register: result
+
+    - name: "End playbook if shellscript is not present"
+      ansible.builtin.fail:
+        msg: "Shellscript to convert markdown to asciidoc is not present or executable"
+      when:
+        - not result.stat.executable
+        - not result.stat.exists
+
+    - name: "Delete markdown file if present"
+      ansible.builtin.file:
+        path: "{{ repo_path }}/guides/hammer_full_help.md"
+        state: absent
+
+    - name: "Verify that foreman-documentation is present"
+      ansible.builtin.git:
+        clone: false
+        dest: "{{ repo_path }}"
+        repo: "{{ repo_remote }}"
+        update: false  # noqa: latest
+        version: "HEAD"
+
+- name: "Generate Hammer CLI help"
+  hosts: foreman-katello
+
+  tasks:
+    - name: "Update DNF cache"
+      ansible.builtin.dnf:
+        update_cache: true
+
+    - name: "Install all available Hammer CLI packages"
+      ansible.builtin.package:
+        name: "rubygem-hammer_cli*"
+        state: latest  # noqa: package-latest
+
+    - name: "Generate full Hammer CLI help"
```

Comment:
This task fails for my instance with the following warnings:
```
"stderr": "Warning: An error occurred while loading module hammer_cli_foreman_ansible.
Warning: An error occurred while loading module hammer_cli_foreman_remote_execution.
Warning: An error occurred while loading module hammer_cli_foreman_tasks.
Error: Resource 'discovered_hosts' does not exist in the API", 
```
I have run it on Foreman without Katello. But `hammer_cli_foreman_ansible` shouldn't require Katello, right?

Reply:
I'm going to retry with a proper instance :))

Reply:
It's completely untested on Foreman on EL or Foreman on Deb. The foreman_ansible plugin does not require Katello AFAIK: https://github.com/theforeman/foreman-packaging/blob/rpm/develop/packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec

For now, I'd accept the limitation that it's Katello/Satellite/orcharhino-only.